site/content/faq/900-is-there-a-router.md

```diff
@@ -4,10 +4,10 @@ question: Is there a router?
 
 The official routing library is [SvelteKit](https://kit.svelte.dev/), which is currently in beta. SvelteKit provides a filesystem router, server-side rendering (SSR), and hot module reloading (HMR) in one easy-to-use package. It shares similarities with Next.js for React.
 
-However, you can use any router lib you want. A lot of people use [page.js](https://github.com/visionmedia/page.js). There's also [navaid](https://github.com/lukeed/navaid), which is very similar. And [universal-router](https://github.com/kriasoft/universal-router), which is similar as well, but with the concept of child routes.
+However, you can use any router lib you want. A lot of people use [page.js](https://github.com/visionmedia/page.js). There's also [navaid](https://github.com/lukeed/navaid), which is very similar. And [universal-router](https://github.com/kriasoft/universal-router), which is isomorphic with child routes, but without built-in history support.
 
-If you prefer a declarative HTML approach, there's [svelte-routing](https://github.com/EmilTholin/svelte-routing).
+If you prefer a declarative HTML approach, there's the isomorphic [svelte-routing](https://github.com/EmilTholin/svelte-routing) library.
```

Comment:
Maybe it's worth mentioning [svelte-navigator](https://github.com/mefechoel/svelte-navigator) too. My experience with `svelte-routing` in the past was that it lacks certain functionality and flexibility. `svelte-navigator`, which started as a fork of it, had covered a lot of advanced usecases. Though I'm fine if we don't mention that too.

Reply:
```suggestion
If you prefer a declarative HTML approach, there's the isomorphic [svelte-routing](https://github.com/EmilTholin/svelte-routing) library and a fork of it called [svelte-navigator](https://github.com/mefechoel/svelte-navigator) containing some additional functionality.
```